docs/includes/google-credentials.md

```diff
@@ -2,12 +2,12 @@ Tip: If you see Google Play Console or Google Developer Console in your local la
 
 1. Open the [Google Play Console](https://play.google.com/apps/publish/)
 1. Click the **Settings** menu entry, followed by **API access**
-1. Click the **CREATE SERVICE ACCOUNT** button 
+1. Click the **CREATE SERVICE ACCOUNT** button
 1. Follow the **Google Developers Console** link in the dialog, which opens a new tab/window:
    1. Click the **CREATE SERVICE ACCOUNT** button at the top of the **Google Developers Console**
    1. Provide a `Service account name`
    1. Click **Select a role** and choose **Service Accounts > Service Account User**
-   1. Check the **Furnish a new private key** checkbox
+   1. Click the **Create key** button
```

Comment:
Did you double check if this is really how it works? 
I think I had trouble confirming the PR that was coming in back then.

Reply:
Very good point - I hadn't 🙈 
Just did, though, and made several adjustments. Also edited the PR description 🙏 

Thanks for pointing this out! I was a fool! 🙇

Reply:
Don't worry, this UI is effectively why that other PR never got merged - it was just to hard to figure out what exactly was going on :D